src/raftstore/store/snap.rs

```diff
@@ -1316,6 +1333,10 @@ impl SnapManager {
             receiving_count: receiving_cnt,
         }
     }
+
+    pub fn get_limiter(&self) -> &Arc<RateLimiter> {
```

Comment:
remove &

Reply:
please take look at this pr https://github.com/pingcap/rust-rocksdb/pull/153, I think this is where goes wrong.

Reply:
Use Arc<RwLock<RateLimiter>>